lib/permission_handler.dart

```diff
@@ -5,19 +5,34 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
 
 part 'package:permission_handler/permission_enums.dart';
 part 'package:permission_handler/utils/codec.dart';
 
 /// Provides a cross-platform (iOS, Android) API to request and check permissions.
 class PermissionHandler {
-  static const MethodChannel _channel =
-      const MethodChannel('flutter.baseflow.com/permissions/methods');
+  factory PermissionHandler() {
+    if (_instance == null) {
+      final MethodChannel methodChannel =
```

Comment:
@martijn00, the static code analyses is complaining about this line and mentions it is better to use `const` over `final` (which is correct):

"info • Prefer const over final for declarations • lib/permission_handler.dart:17:7 • prefer_const_declarations"

So could you change this to:

```
const MethodChannel methodChannel =
    const MethodChannel('flutter.baseflow.com/permissions/methods');
```